lib/main_micro_alp.pm

```diff
@@ -119,6 +114,7 @@ sub load_installation_tests {
         }
         loadtest 'console/textinfo';
     }
+    replace_opensuse_repos_tests if is_repo_replacement_required;
```

Comment:
Shouldn't this happen before `transactional/tdup`? :thinking:

Reply:
Yes, and it does.

Reply:
This one applies to only installations, so the tdup would be schedule afterwards as `load_config_tests` are runs after installation

Reply:
ok